cores/nRF5/sysview/SEGGER/SEGGER_RTT.h

```diff
@@ -150,6 +204,7 @@ unsigned     SEGGER_RTT_HasDataUp               (unsigned BufferIndex);
 void         SEGGER_RTT_Init                    (void);
 unsigned     SEGGER_RTT_Read                    (unsigned BufferIndex,       void* pBuffer, unsigned BufferSize);
 unsigned     SEGGER_RTT_ReadNoLock              (unsigned BufferIndex,       void* pData,   unsigned BufferSize);
+int          SEGGER_RTT_Peek                    (void);

```

Comment:
we don't need to have RTT as Stream instance, we can change nanolib `_write()` hook to retarget to rtt.